defaults/main.yml

```diff
@@ -23,7 +26,7 @@ grafana_conf_grafana_datasource_definitions:
     "user": "",
     "password": "",
     "type": "monasca-datasource",
-    "url": "http://{{ grafana_conf_monasca_api.host }}:{{ grafana_conf_monasca_api.port }}/",
+    "url": "http://{{ item.value.host }}:{{ item.value.port }}/",
```

Comment:
For a small (optional) usability enhancement you could set a task variable, then reference that instead of `item.host`.

```
vars:
  datasource: "{{ item.value }}"

Reply:
Good suggestion - fixed.